### PR TITLE
Add annotations block to helm chart

### PR DESCRIPTION
**Pull request by almightykid**
I've taken the initiative to update the chart and ensure its continued maintenance across future releases. The necessary adjustments have been implemented and I would like to submit a pull request (PR) to make these updates accessible to everyone.

### CODE REVIEW

File: packaging/helm-charts/helm3/strimzi-drain-cleaner/templates/_helpers.tpl [hunk 1]
```diff
@@ -51,6 +51,13 @@ Selector labels
 app: {{ include "strimzi-drain-cleaner.name" . }}
 {{- end }}
 
+{{/*
+Annotations
+*/}}
+{{- define "strimzi-drain-cleaner.annotations" -}}
+app: {{ include "strimzi-drain-cleaner.name" . }}
```
Comment (scholzj) [inline packaging/helm-charts/helm3/strimzi-drain-cleaner/templates/_helpers.tpl:58]: Why should we have this annotation there? I would say it should be empty by default.

File: packaging/helm-charts/helm3/strimzi-drain-cleaner/values.yaml [hunk 2]
```diff
@@ -70,6 +70,8 @@ podLabels: {}
   # Specifies optional custom labels for Pod.
   # If you want to add custom labels, add them below and remove curly braces above.
 
+annotations: {}
```
Comment (scholzj) [inline packaging/helm-charts/helm3/strimzi-drain-cleaner/values.yaml:73]: You are adding a field for pod annotations -> I think that is fine if there is some value for it. But it should be clearly named as `podAnnotations`.

**Review by scholzj — COMMENTED** [review]
Thanks for the PR. I left some comments. 

Also, please make sure to fix the DCO signoff. The instructions should be under the Details link in the Pr status section.

File: packaging/helm-charts/helm3/strimzi-drain-cleaner/templates/_helpers.tpl (hunk 1 above)
Comment (almightykid) [reply packaging/helm-charts/helm3/strimzi-drain-cleaner/templates/_helpers.tpl]: I have submitted a new commit with this removed as you are right.

File: packaging/helm-charts/helm3/strimzi-drain-cleaner/values.yaml (hunk 2 above)
Comment (almightykid) [reply packaging/helm-charts/helm3/strimzi-drain-cleaner/values.yaml]: Updated in latest commit 👍

**Comment by almightykid** [comment]
Let me see how to fix the DCO signoff

**Comment by scholzj** [comment]
> Also, please make sure to fix the DCO signoff. The instructions should be under the Details link in the Pr status section.

As I said, ` The instructions should be under the Details link in the Pr status section.`.

**Review by scholzj — APPROVED** [review]
LGTM. Thanks.

**Review by im-konge — APPROVED** [review]

**Review by see-quick — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**